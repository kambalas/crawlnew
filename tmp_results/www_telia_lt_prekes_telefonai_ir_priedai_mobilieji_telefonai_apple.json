[
  {
    "name": "Apple iPhone 16e",
    "price": "609",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro",
    "price": "1028",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max",
    "price": "1269",
    "error": false
  },
  {
    "name": "Apple iPhone 16",
    "price": "769",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus",
    "price": "909",
    "error": false
  },
  {
    "name": "Apple iPhone 15",
    "price": "759",
    "error": false
  },
  {
    "name": "Apple iPhone 12",
    "price": "459",
    "error": false
  },
  {
    "name": "Apple iPhone 13",
    "price": "499",
    "error": false
  }
]