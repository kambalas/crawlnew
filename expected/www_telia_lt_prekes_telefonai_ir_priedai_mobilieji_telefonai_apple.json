[
  {
    "name": "Apple iPhone 16e 128 gb skaidri",
    "price": "609",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 128 gb juoda",
    "price": "1 028",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 256 gb juoda",
    "price": "1 269",
    "error": false
  },
  {
    "name": "Apple iPhone 16 128 gb juoda",
    "price": "769",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus 128 gb juoda",
    "price": "909",
    "error": false
  },
  {
    "name": "Apple iPhone 15",
    "price": "739",
    "error": false
  },
  {
    "name": "Apple iPhone 13 128 gb juoda",
    "price": "499",
    "error": false
  },
  {
    "name": "Apple iPhone 12 64 gb juoda",
    "price": "459",
    "error": false
  }
]